### ISSUE TITLE
Bad background color after resurrection?

### ISSUE THREAD

**Issue by davidfmatheson** [catppuccin/tmux#327]
### Is there an existing issue outlining your problem?

- [X] I have searched the existing issues and they do not solve my problem.

### Describe your problem.

Whenever my tmux sessions are recovered via resurrect, the background colors look like this:
<img width="567" alt="image" src="https://github.com/user-attachments/assets/74133571-742b-493e-ac42-9f17967106ca">

I bookmark Issue #180 just so I can go back to this workaround, loading @guedesfelipe's fork
```sh
set -g @plugin 'guedesfelipe/catppuccin_tmux#1.0.0'
```

and running `<prefix>-I`, which flips it back
<img width="459" alt="image" src="https://github.com/user-attachments/assets/9cd1166c-bf56-4c83-ac46-4d732f7b47c2">

Going back to `set -g @plugin 'catppuccin/tmux#latest'` it remains good, until the next time it is resurrected.

### Paste your configuration.

```sh
set -g mouse on
bind -r S-Left swap-window -t -1\; select-window -t -1
bind -r S-Right swap-window -t +1\; select-window -t +1
set -g status-position top
# Catppuccin setup
set -g @plugin 'catppuccin/tmux#latest'
set -g @catppuccin_window_left_separator ""
set -g @catppuccin_window_right_separator " "
set -g @catppuccin_window_middle_separator " █"
set -g @catppuccin_window_number_position "right"
set -g @catppuccin_window_default_fill "number"
set -g @catppuccin_window_default_text "#W"
set -g @catppuccin_window_current_fill "number"
set -g @catppuccin_window_current_text "#W#{?window_zoomed_flag,(),}"
set -g @catppuccin_status_modules_right "directory date_time battery"
set -g @catppuccin_status_modules_left "session"
set -g @catppuccin_status_left_separator  " "
set -g @catppuccin_status_right_separator " "
set -g @catppuccin_status_right_separator_inverse "no"
set -g @catppuccin_status_fill "icon"
set -g @catppuccin_status_connect_separator "no"
set -g @catppuccin_directory_text "#{pane_current_path}"
set -g @catppuccin_date_time_text "%H:%M"
set -g @plugin 'tmux-plugins/tmux-battery'
# Continuum setup
set -g @plugin 'tmux-plugins/tmux-resurrect'
set -g @plugin 'tmux-plugins/tmux-continuum'
set -g @continuum-restore 'on'
# Vim tmux navigator
set -g @plugin 'christoomey/vim-tmux-navigator'
run '~/.config/tmux/plugins/tpm/tpm'
```

### What tmux version are you seeing the issue on?

tmux 3.5

**Comment by kjnsn** [comment]
Hmmm interesting. Can you try with `set -g @plugin 'catppuccin/tmux#latest'` (the main branch)? Otherwise you are welcome to pin to `set -g @plugin 'catppuccin/tmux#v0.3.0'`, which should maintain compatibility with your configuration

**Comment by davidfmatheson** [comment]
Generally I roll with `set -g @plugin 'catppuccin/tmux#latest'`. I am pinning to 0.3.0 right now given Issue #330, and it still gives me those wonky backgrounds after a resurrection.  I tried 0.2.0 as well, same issue. I suspect it had something to do with Issue #180, so it's been present for a while.

**Comment by davidfmatheson** [comment]
> This _might_ get fixed by #336

Happy to try it. Can you let me know how to point to a PR?

**Comment by kjnsn** [comment]
> > This _might_ get fixed by #336
> 
> Happy to try it. Can you let me know how to point to a PR?

This has been merged, can you see if it still reproduces? Ensuring that you are using the latest plugin version, and everything is using a brand new tmux server

**Comment by davidfmatheson** [comment]
It's hard to tell when I can't get my standard configuration back to working after updating to latest, but it seems like it's still broken. Here is a resurrected session:
<img width="668" alt="image" src="https://github.com/user-attachments/assets/b67d9d09-ab45-4c19-8872-5715812858b3">

and here is what it looks like after navigating to each window to reset the background colors:
<img width="639" alt="image" src="https://github.com/user-attachments/assets/d495b4ef-677d-4fc0-9086-939f3a38bd85">
